sdk/tests/conformance2/extensions/ext-color-buffer-float.html

```diff
@@ -193,37 +193,63 @@
 function runFloatRenderbufferRenderTargetTest(enabled, internalFormat, testProgram, numberOfChannels, subtractor)
 {
     var formatString = wtu.glEnumToString(gl, internalFormat);
-    debug("");
-    debug("testing floating-point " + formatString + " renderbuffer render target");
-
-    var colorbuffer = gl.createRenderbuffer();
-    var width = 2;
-    var height = 2;
-    gl.bindRenderbuffer(gl.RENDERBUFFER, colorbuffer);
-    gl.renderbufferStorage(gl.RENDERBUFFER, internalFormat, width, height);
-    if (!enabled) {
-        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, "floating-point renderbuffer allocation should fail if EXT_color_buffer_float is not enabled");
-        return;
-    } else {
-        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "floating-point renderbuffer allocation should succeed if EXT_color_buffer_float is enabled");
+    var samples = [0];
+    if (enabled) {
+        samples = Array.prototype.slice.call(gl.getInternalformatParameter(gl.RENDERBUFFER, internalFormat, gl.SAMPLES));
+        samples.push(0);
     }
+    for (var ndx = 0; ndx < samples.length; ++ndx) {
+        debug("");
+        debug("testing floating-point " + formatString + " renderbuffer render target with number of samples " + samples[ndx]);
+
+        var colorbuffer = gl.createRenderbuffer();
+        var width = 2;
+        var height = 2;
+        gl.bindRenderbuffer(gl.RENDERBUFFER, colorbuffer);
+        gl.renderbufferStorageMultisample(gl.RENDERBUFFER, samples[ndx], internalFormat, width, height);
```

Comment:
Although according to the spec, renderbufferStorageMultisample(samples == 0) is the same as renderbufferStorage, but we still should verify the implementation follows the spec.  I suggest we test both renderbufferStorageMultisample(samples == 0) and renderbufferStorage